integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java

```diff
@@ -17,7 +32,61 @@ public class AbstractWebIT extends AbstractIT {
   protected static WebDriver driver;
 
   // https://www.selenium.dev/documentation/webdriver/waits/#implicit-waits
-  protected static final long MAX_IMPLICIT_WAIT = 10;
+  protected static final long MAX_IMPLICIT_WAIT = 30;
+  protected static final long MAX_TIMEOUT = 20;
+  protected static final long ACTION_SLEEP_MILLIS = 1_000;
+
+  protected boolean waitShowText(final String text, final By locator) {
+    try {
+      WebElement element = pollingWait(locator, MAX_TIMEOUT);
+      return text.equals(element.getText());
+    } catch (TimeoutException e) {
+      return false;
+    }
+  }
+
+  protected WebElement pollingWait(final By locator, final long maxTimeout) {
+    Wait<WebDriver> wait =
+        new FluentWait<>(driver)
+            .withTimeout(Duration.of(maxTimeout, ChronoUnit.SECONDS))
+            .pollingEvery(Duration.of(1, ChronoUnit.SECONDS))
+            .ignoring(NoSuchElementException.class);
+
+    return wait.until((Function<WebDriver, WebElement>) driver -> driver.findElement(locator));
+  }
+
+  protected void clickAndWait(final Object locatorOrElement) throws InterruptedException {
+    WebElement element;
+    if (locatorOrElement instanceof By) {
+      element = pollingWait((By) locatorOrElement, MAX_IMPLICIT_WAIT);
+    } else if (locatorOrElement instanceof WebElement) {
+      element = (WebElement) locatorOrElement;
+    } else {
+      throw new InvalidArgumentException("The provided argument is neither a By nor a WebElement");
+    }
+    try {
+      WebDriverWait wait = new WebDriverWait(driver, MAX_TIMEOUT);
+      wait.until(ExpectedConditions.visibilityOf(element));
+      wait.until(ExpectedConditions.elementToBeClickable(element));
+
+      element.click();
+      Thread.sleep(ACTION_SLEEP_MILLIS);
+    } catch (ElementClickInterceptedException e) {
+      Actions action = new Actions(driver);
+      action.moveToElement(element).click().build().perform();
+      Thread.sleep(ACTION_SLEEP_MILLIS);
+      LOG.error(e.getMessage(), e);
+    }
+  }
+
+  @BeforeEach
+  public void beforeEachTest() {
+    try {
+      Thread.sleep(ACTION_SLEEP_MILLIS);
```

Comment:
I think better change variable name `ACTION_SLEEP_MILLIS` to `EACH_TEST_SLEEP_MILLIS`

Reply:
Done